xamarin/nunit.tests.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:installLocation="auto">
-	<uses-sdk android:minSdkVersion="15" />
+	<uses-sdk android:minSdkVersion="15" android:targetSdkVersion="27" />
```

Comment:
What does this do?

Reply:
Not sure, came in when I updated.  Will remove it.